pkg/misc/depversion/depversion_test.go

```diff
@@ -485,16 +485,3 @@ func Test_DoesRangeInclude(t *testing.T) {
 		})
 	}
 }
-
-func Test_DoesRangeInclude_Errors(t *testing.T) {
```

Comment:
hey @rmetzman, can you rebase instead of merging the 'main' branch into fixRangeTest.